VictoryCenter/VictoryCenter.UnitTests/ValidatorsTests/Partners/BasePartnerSectionValidatorTests.cs

```
@@ -30,24 +29,52 @@ public BasePartnerSectionValidatorTests()
     {
         _validator = new TestBasePartnerSectionValidator();
 
-        _validTitle = "Valid Title";
+        _validTitle = new string('T', PartnerConstants.PartnersSectionTitleMinLength);
+        _tooShortTitle = new string('T', PartnerConstants.PartnersSectionTitleMinLength > 0 ?
+            PartnerConstants.PartnersSectionTitleMinLength - 1 : 0);
         _tooLongTitle = new string('T', PartnerConstants.PartnersSectionTitleMaxLength + 1);
 
-        _validDescription = "A valid section description.";
+        _validDescription = new string('D', PartnerConstants.PartnersSectionDescriptionMinLength);
+        _tooShortDescription = new string('D', PartnerConstants.PartnersSectionDescriptionMinLength > 0 ?
+            PartnerConstants.PartnersSectionDescriptionMinLength - 1 : 0);
         _tooLongDescription = new string('D', PartnerConstants.PartnersSectionDescriptionMaxLength + 1);
     }
 
+    [Theory]
+    [InlineData(null)]
+    [InlineData("")]
+    public void Validate_TitleIsEmptyOrNull_ShouldHaveError(string title)
+    {
+        // Arrange
+        var model = new TestPartnerSectionDto { Title = title, Description = _validDescription };
+
+        // Act
+        var result = _validator.TestValidate(model);
+
+        // Assert
+        result.ShouldHaveValidationErrorFor(x => x.Title)
+              .WithErrorMessage(ErrorMessagesConstants.PropertyIsRequired(nameof(BasePartnerSectionCreateUpdateDto.Title)));
+    }
+
     [Fact]
-    public void Validate_TitleIsEmpty_ShouldHaveError()
+    public void Validate_TitleIsTooShort_ShouldHaveError()
     {
         // Arrange
-        var model = new TestPartnerSectionDto { Title = "" };
+        // Skip test if MinLength is 0 or 1, as "" (NotEmpty) would catch it.
```

1. Why are doing checks, if your data is already predefined? 2. This comment is wrong, NotEmpty catches only empty strings, string of length 1 is valid.